diff -urN ckb-next-0.6.2/CMakeLists.txt ckb-next-0.6.2-cmakelist-qt6/CMakeLists.txt
--- ckb-next-0.6.2/CMakeLists.txt	2025-03-17 17:54:36.000000000 +0000
+++ ckb-next-0.6.2-cmakelist-qt6/CMakeLists.txt	2025-05-10 20:51:58.578078157 +0100
@@ -158,9 +158,11 @@
 set(CMAKE_FIND_PACKAGE_SORT_ORDER NATURAL)
 set(CMAKE_FIND_PACKAGE_SORT_DIRECTION DEC)
 
-set(QT_LIST Qt5)
+
 if (PREFER_QT6)
-    list(INSERT QT_LIST 0 Qt6)
+    set(QT_LIST Qt6)
+else()
+    set(QT_LIST Qt5)
 endif ()
 
 if (NOT WITH_GUI)
